src/main/java/com/mineboundteam/minebound/magic/DefensiveSpells/TelekineticDefensiveSpell.java

```diff
@@ -0,0 +1,199 @@
+package com.mineboundteam.minebound.magic.DefensiveSpells;
+
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+import com.mineboundteam.minebound.MineBound;
+import com.mineboundteam.minebound.config.IConfig;
+import com.mineboundteam.minebound.item.armor.ArmorTier;
+import com.mineboundteam.minebound.magic.ActiveSpellItem;
+import com.mineboundteam.minebound.magic.MagicType;
+import com.mineboundteam.minebound.magic.SpellType;
+import com.mineboundteam.minebound.magic.events.MagicSelectedEvent;
+
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.damagesource.DamageSource;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.entity.MoverType;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.entity.projectile.ProjectileUtil;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.phys.AABB;
+import net.minecraft.world.phys.EntityHitResult;
+import net.minecraft.world.phys.Vec3;
+import net.minecraftforge.common.ForgeConfigSpec;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.config.ModConfigEvent;
+
+@Mod.EventBusSubscriber(modid = MineBound.MOD_ID)
+public class TelekineticDefensiveSpell extends ActiveSpellItem {
+
+    protected final TelekineticDefensiveSpellConfig config;
+
+    public TelekineticDefensiveSpell(Properties properties, TelekineticDefensiveSpellConfig config) {
+        super(properties, config.LEVEL, MagicType.TELEKINETIC, SpellType.DEFENSIVE);
+        this.config = config;
+    }
+
+    protected static final ConcurrentHashMap<Integer, Integer> selectedMap = new ConcurrentHashMap<>();
+    protected static final ConcurrentHashMap<Integer, InteractionHand> attackMap = new ConcurrentHashMap<>();
+
+    @Override
+    public void use(ItemStack stack, InteractionHand usedHand, Level level, Player player) {
+        if(!level.isClientSide()) {
+            Integer id = selectedMap.get(player.getId());
+            if(id != null){
+                Entity entity = level.getEntity(id);
+                if(entity != null){
+                    attackMap.put(player.getId(), usedHand);
+                    entity.hurt(DamageSource.playerAttack(player).setIsFall(), (float)(config.DAMAGE.get().doubleValue()));
+                    reduceMana((int)(config.MANA_COST.get() * 10), player);
+                }
+            } else {
```

Comment:
`id` is always `null` in my testing, so the squeeze damage is never applied.

Reply:
Do you have the spell equipped with both hands? It will only apply the damage if you're already holding something with your other hand

Reply:
Got it. I missed that upon reading the description the first time.

Reply:
Maybe change the wording of that part of the description to make it clear you need to use two of the spells in tandem to apply the squeeze damage. Something like "If a mob is already being held, will instead squeeze them for X damage"

Reply:
Updated